lib/PuppeteerSharp/Page.cs

```diff
@@ -848,8 +852,9 @@ private void EmitMetrics(PerformanceMetricsResponse metrics)
             }
         }
 
-        private void HandleException(dynamic exceptionDetails)
+        private void HandleException(PageError pageError)
         {
+            PageError?.Invoke(this, new PageErrorEventArgs(pageError));
```

Comment:
I noticed that puppeteer calls `helper.getExceptionMessage(exceptionDetails)` and then raises the event with a simple exception that only contains a message. not the complete `exceptionDetails`

Reply:
Agreed, I went too far there.
I'll implement a RuntimeExceptionThrownResponse with only the data we need there, but I would still need a class called "Exception"...

Reply:
`ExceptionData`?

Reply:
maybe just a `Message` property?

Reply:
can't we just stick it on `PageErrorEventArgs`?